src/frontend/SettingsDialog.ui

```diff
@@ -13,132 +13,139 @@
   <property name="windowTitle">
```

Comment:
![settings_flaw](https://user-images.githubusercontent.com/9459891/43338196-82e2bc6a-91f7-11e8-873f-c1668c43f07e.png)
The padding didn't work. There are now two lines which are being cut off.

Reply:
Hmm, That's weird. This window supposes to have a fixed width. It is showing way smaller in your screenshot.

Reply:
@mominul Can you come to discord.